src/js/views/patients/patient/patient.scss

```diff
@@ -151,3 +151,33 @@
   padding: 0 8px;
   vertical-align: middle;
 }
+
+.patient-flow__progress {
```

Comment:
I think the progress bar styles can be moved into a `/module` with the exception of `width` `margin-top` or any other thing that is likely context specific.